### ISSUE TITLE
Release 2.0.0 breaks plugin page with JS error

### ISSUE THREAD

**Issue by aleho** [performancecopilot/grafana-pcp#34]
Navigating to the plugin page of grafana-pcp (to activate it) causes a JS error and infinite loading spinner:

```
Error: define is not a function
  Evaluating https://grafana/public/plugins/performancecopilot-pcp-app/module.js
  Loading plugins/performancecopilot-pcp-app/module
```

Plugin was installed via docker env `GF_INSTALL_PLUGINS: "https://github.com/performancecopilot/grafana-pcp/archive/v2.0.0.zip;grafana-pcp"`.

**Comment by andreasgerstmayr** [comment]
Thanks a lot for the bug report! Much appreciated.

Looks like there was a change in the minification process for production builds, and systemjs couldn't parse the plugin anymore (see systemjs/systemjs#2117 and grafana/grafana#21785).

I've implemented a workaround and released version 2.0.1.

`$ podman run -e GF_INSTALL_PLUGINS="https://github.com/performancecopilot/grafana-pcp/archive/v2.0.1.zip;grafana-pcp" -p 3000:3000 grafana/grafana` works fine now for me - can you check and report back?

**Comment by aleho** [comment]
Thanks for the fast fix, 2.0.1 works fine for me!
